src/Engine/Airspace/AbstractAirspace.hpp

```diff
@@ -244,6 +244,16 @@ class AbstractAirspace {
     return astype;
   }
 
+    /**
+     * Get Class or Type of airspace
```

Comment:
Docstring along the lines of:
 * Returns the airspace class type. If `GetType` is `AirspaceClass::OTHER`, 
 * returns `GetClass()`, otherwise returns `GetType()`.
 *
 * @return AirspaceClass - The determined airspace class type.

Reply:
Done.